perception/traffic_light_multi_camera_fusion/src/node.cpp

```diff
@@ -0,0 +1,291 @@
+// Copyright 2023 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "traffic_light_multi_camera_fusion/node.hpp"
+
+#include <algorithm>
+#include <memory>
+#include <string>
+#include <vector>
+
+namespace
+{
+
+bool isUnknown(const autoware_auto_perception_msgs::msg::TrafficSignal & signal)
+{
+  return signal.lights.size() == 1 &&
```

Comment:
@Mingyu1991 
Maybe I am wrong but shouldn't either the color or shape being unknown make the signal unknown?

Reply:
@knzo25 
Based on the implementation [here](https://github.com/autowarefoundation/autoware.universe/blob/main/perception/traffic_light_classifier/src/cnn_classifier.cpp#L205), both shape and color will be makred unknown.